editor/data-binding/overview.mdx

```diff
@@ -119,6 +122,8 @@ View model properties can represent more types of data compared to inputs and ev
 | Colors                 | ❌     | ❌     | ✅                     |
 | Nesting                | ❌     | ❌     | ✅                     |
 
+We intend to support lists and images in following releases.
```

Comment:
We could consider putting these in the table and saying: "Coming soon". But not a blocker and up for discussion